scaling_performance/optimizing_compute_resources.adoc

```diff
@@ -0,0 +1,87 @@
+[[scaling-performance-compute-resources]]
+= Optimizing Compute Resources
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+
+[[scaling-performance-overcomitting]]
+== Overcommitting
+
+You can use overcommit procedures so that resources such as CPU and memory are
+more accessible to the parts of your cluster that need them.
+
+Note that when you overcommit, there is a risk that another application may
+not have access to the resources it requires when it needs them, which will
+result in reduced performance. However, this may be an acceptable tradeoff in
+favor of increased density and reduced costs. For example, development, QA or
+test environments may be overcommited, whereas production might not be.
+
+{product-title} implements resource management through the compute resource model and
+quota system. See the documentation for more information about the
+xref:../dev_guide/compute_resources.html[OpenShift resource model].
+
+For more information and strategies for overcommiting, see the
+xref:../admin_guide/overcommit.html[Overcommitting documentation in the Cluster
+Administration Guide].
+
+[[scaling-performance-image-considerations]]
+== Image Considerations
+
+[[scaling-performance-gold-image]]
+=== Using a Gold Image to Improve Efficiency
+
+You can create a base {product-title} image with a number of tasks built-in to
```

Comment:
I suppose gold image is a term we use on our team, and image-based deployments would be more of a known term in the industry, so let's go with that please.